core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -482,11 +484,8 @@ oppia.directive('conversationSkin', [
           };
 
           $scope.submitAnswer = function(answer, interactionRulesService) {
-            // For some reason, answers are getting submitted twice when the
-            // submit button is clicked. This guards against that.
-            if (_answerIsBeingProcessed ||
-                !$scope.isCurrentCardAtEndOfTranscript() ||
-                $scope.activeCard.destStateName) {
+            if (fatigueDetectionService.addSubmission()) {
+              $scope.$broadcast('oppiaFeedbackAvailable');
               return;
             }
```

Comment:
Wasn't sure what the original person was referring to. But I did notice that on TextInput and NumericInput double-submissions were happening. Maybe that was it? After I removed the ng-click on the buttons it correctly only sent one submission.